### ISSUE TITLE
Is "CIFAR_load" module missing in this repository?

### ISSUE THREAD

**Issue by Szy-Young** [RobRomijnders/RAM#1]
Hi! Thanks for sharing your model and I'd like to train on dataset CIFAR10 with "RAM_cifar_main.py" in your repository. However it calls a module named "CIFAR_load" which cannot be found. I'm afraid you forgot to package this loading-and-preprocessing module with other files...
If so could you please add it to this repository? Or if it comes from other work, could you please show the source?

**Comment by RobRomijnders** [comment]
That's right. The original source is [here](https://github.com/seann999/tensorflow_mnist_ram)
